optionals/nouniformrestrictions/functions/fnc_exportConfig.sqf

```diff
@@ -0,0 +1,52 @@
+/*
+ * Author: BaerMitUmlaut, 654wak654
+ * Generates the CfgVehicles.hpp to unlock all uniforms.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * CfgVehicles.hpp content <STRING>
+ *
+ * Example:
+ * [] call ace_nouniformrestrictions_fnc_exportConfig
+ *
+ * Public: [Yes]
+ */
+#include "script_component.hpp"
+
+private _modifyClasses = [];
+private _baseClasses = [];
+{
+    private _modifyClass = {
+        if (!isNull (_x >> "modelSides")) exitWith {_x};
+    } forEach (configHierarchy _x);
+    private _baseClass = inheritsFrom _modifyClass;
+    _modifyClasses pushBackUnique [_modifyClass, _baseClass];
+    if !(_baseClass in (_modifyClasses apply {_x select 0})) then {
+        _baseClasses pushBackUnique _baseClass;
+    };
+    false
+} count (
+    ("!isNull (_x >> 'modelSides') &&" +
+    "{!(getArray (_x >> 'modelSides') isEqualTo [6])} &&" +
+    "{!(getArray (_x >> 'modelSides') isEqualTo [0,1,2,3])}")
+    configClasses (configFile >> "CfgVehicles")
```

Comment:
Can we filter out uniforms that inherit their `modelSides` value from their base class?